Frontend/mobile/app/services/profile.service.ts

```
@@ -0,0 +1,91 @@
+import api from '../utils/api';
+import { User, AuthError } from '../types/auth';
+
+class ProfileService {
+  /**
+   * Fetch user profile
+   */
+  async getProfile(): Promise<User> {
+    try {
+      const response = await api.get<User>('/authenticator/profile/');
+      return response.data;
+      console.log(response.data);
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)